docs/docs/cmd/teams/message/message-get.md

```diff
@@ -32,3 +32,83 @@ Retrieve the specified message from a channel of the Microsoft Teams team
 ```sh
 m365 teams message get --teamId 5f5d7b71-1161-44d8-bcc1-3da710eb4171 --channelId 19:88f7e66a8dfe42be92db19505ae912a8@thread.skype --id 1540747442203
 ```
+
+## Response
+
+=== "JSON"
+
+    ``` json
+    {
+      "id": "1666799520731",
```

Comment:
Let's try to align the output with the data used in the example. Same goes for output types text and csv and for the variables teamId and channelId